Add INPUT_REPOSITORY for another repository push (#7)
ad-m/github-push-action: README.md

| name | value | default | description |
| ---- | ----- | ------- | ----------- |

> | github_token | string | | Token for the repo. Can be passed in using `{{ secrets.GITHUB_TOKEN }}`. |

Reviewer: Not sure about this, and this is a bug from the past, but shouldn't this be `${{ <expression> }}`?

| branch | string | 'master' | Destination branch to push changes. |
| force | boolean | false | Determines if force push is used. |
| directory | string | '.' | Directory to change to before pushing. |

> | repository | string | '' | Repository name. Default or empty repository name represents current github repository. If you want to push other repository, you should make a personal access token(https://github.com/settings/tokens) and put it with github_token.  |

Reviewer: ```suggestion
| repository | string | '' | Repository name. Default or empty repository name represents current github repository. If you want to push to other repository, you should make a [personal access token](https://github.com/settings/tokens) and use it as the `github_token` input.  |
```

Author: @tenhobi Thanks a lot! :->  I modified and pushed it.
